Maskbook/Scene/Wallet/Balance/View/AccountCardView.swift

```diff
@@ -425,24 +425,42 @@ class AccountCardView: UIView {
             balanceLabel.text = "\(maskUserDefaults.currency.symbol)\(totalBalance.currency)"
         }
         if let account = account {
-            Publishers.CombineLatest4(
-                account.publisher(for: \.name).eraseToAnyPublisher(),
-                account.publisher(for: \.address).eraseToAnyPublisher(),
-                UserDefaultSettings.shared.displayBlockChainPublisher.eraseToAnyPublisher(),
-                UserDefaultPublishers.network.eraseToAnyPublisher()
-            )
-            .receive(on: DispatchQueue.main)
-            .sink { [weak self] name, address, displayBlockChain, networkType in
-                guard let validAddress = address else { return }
-                self?.updateBlockChainButtonStatus(currentBlockChain: displayBlockChain)
-                self?.nameLabel.text = name
-                self?.addressLabel.text = "\(validAddress.prefix(10))...\(validAddress.suffix(10))"
-                self?.networkLabel.text = networkType.shortName.lowercased().capitalized
-                self?.networkIcon.image = networkType.smallIcon?.withTintColor(Asset.Colors.AccountCard.nameText.color)
-                self?.updateBackground(isWalletConnect: account.fromWalletConnect,
-                                       displayBlockchain: displayBlockChain)
-            }
-            .store(in: &disposeBag)
+            UserDefaultSettings.shared.displayBlockChainPublisher.eraseToAnyPublisher()
```

Comment:
Maybe implement  `UserDefaultSettings` dependency in `AccountCardView` by:
```
@InjectedProvider(\.userDefaultSettings)
private var userDefaultSettings
```

And then refine the code to: 
```
userDefaultSettings.displayBlockChainPublisher
            .combineLatest(userDefaultSettings.networkPubisher)
```

note `userDefaultSettings.networkPubisher` equals to `UserDefaultPublishers.network`, we should avoid using `UserDefaultPublishers` which is replaced by `UserDefaultSettings`.